### DISCUSSION TITLE
020

### DISCUSSION THREAD

**Discussion by giscus** [zhiyu1998/rrorange-and-friends-weekly#10] [General]
# 020

《大橘和朋友们的周刊》： 分享日常冲浪互联网看到好玩的网站、app应用、资源分享、效率软件工具集等

https://rrorangeandfriends.site/posts/2024/020

**Comment by mumingfang** [comment]
投稿

橘猫起始页

描述 ：一款极简风格的浏览器新标签页插件，无需登录完全免费，支持webdev

网址：http://jvmao.net/

**Reply by zhiyu1998** [reply]
收到，感谢投递

**Comment by suding-china** [comment]
暮光是日落吧
